### PR TITLE
chore(code-style): added usabilla specific idea code style template

### PR DESCRIPTION
**Pull request by spirosikmd**

### CODE REVIEW

File: idea-code-style/usabilla-idea-code-style.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<code_scheme name="usabilla-idea-code-style">
+  <JSCodeStyleSettings>
+    <option name="USE_SEMICOLON_AFTER_STATEMENT" value="false" />
```
Comment (beerecca) [inline idea-code-style/usabilla-idea-code-style.xml]: Hmm really? What's our team preference on semicolons? I thought we liked them... :)

Comment (spirosikmd) [reply idea-code-style/usabilla-idea-code-style.xml]: Oh yes we do. Why is that there? And why my editor is not complaining?

**Comment by beerecca** [comment]
This doesn't do the stuff you mentioned about language defaults, is that possible to add? Maybe possible to also adding wallaby with the right config?
```* language-frameworks -> javascript -> es6
* language-frameworks -> javascript -> libraries -> jasmine-Definitely-Typed
* language-frameworks -> javascript -> code quality tools -> eslint
* language-frameworks -> node and npm -> enable```

**Review by pietvanzoen — CHANGES_REQUESTED** [review]
I believe this is all covered in the [`.editorconfig`](https://github.com/usabilla/js-styleguide/blob/master/node-project-template/.editorconfig). Provided you have the editor config plugin installed these rules apply.

However, we don't have instructions for that. So we should. Probably in the `/node-project-template/README.md`?

**Comment by spirosikmd** [comment]
We should commit these anyway in this project I think. If we want to share other idea project settings we should think of another way. Should we go on and close this PR?

**Comment by pietvanzoen** [comment]
So far everything inside `idea-code-style/usabilla-idea-code-style.xml` is just a duplication of what is in `.editorconfig`. We should be using the `.editorconfig` plugin for all projects that have an `.editorconfig`. And since `.editorconfig` trumps (ugh) the internal editor configuration, setting them in this code style xml actually has no effect.

**Comment by pietvanzoen** [comment]
Wait, are you agreeing? I'm confused. 😛  Yes close unless there's something we want to add/change that isn't in `.editorconfig`

**Comment by pietvanzoen** [comment]
Oh.. maybe add the .editorconfig stuff to the `node-project-template` readme?

Or at least add an issue for it.

**Comment by spirosikmd** [comment]
I do agree! By close I mean abandon this PR. We should add the `.editorconfig` in another PR.

**Comment by spirosikmd** [comment]
I created #16 for the editor config stuff.

**Pull request closed without merging**